tools/create-genesis/Cargo.toml

```diff
@@ -9,14 +9,14 @@ serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
 json = "0.11.13"
 serde_json = "1.0.39"
-ethabi = "7.0.0"
-ethereum-types = "0.5.2"
+ethabi = "6.1.0"
+ethereum-types = "0.4"
```

Comment:
why ethereum types should be `0.4` ? and ethabi should be `6.2.0`

Reply:
Done

Reply:
ethabi 7.0.0 -> ethereum-types 0.5.2
ethabi 6.1.0 -> ethereum-types 0.4

So i dump the ethabi version.